packages/react-core/src/demos/Masthead.md

```diff
@@ -0,0 +1,205 @@
+        this.state.searchValue === ''
+          ? this.items
+          : this.items.filter(str => str.toLowerCase().indexOf(this.state.searchValue.toLowerCase()) !== -1);
+
+      this.setState({ filteredItems: filtered || [] });
+    };
+  }
+
+  render() {
+    const { isOpen, isDropdownOpen, isKebabOpen, selected, searchValue, filteredItems } = this.state;
+
+    const dropdownItems = [
+      <DropdownItem key="link">Link</DropdownItem>,
+      <DropdownItem key="action" component="button">
+        Action
+      </DropdownItem>,
+      <DropdownItem key="disabled link" isDisabled href="www.google.com">
+        Disabled Link
+      </DropdownItem>,
+      <DropdownItem
+        key="disabled action"
+        isAriaDisabled
+        component="button"
+        tooltip="Tooltip for disabled item"
+        tooltipProps={{ position: 'top' }}
+      >
+        Disabled Action
+      </DropdownItem>,
+      <DropdownSeparator key="separator" />,
+      <DropdownItem key="separated link">Separated Link</DropdownItem>,
+      <DropdownItem key="separated action" component="button">
+        Separated Action
+      </DropdownItem>
+    ];
+
+    return (
+      <Masthead id="basic">
+        <MastheadToggle>
+          <Button variant="plain" onClick={() => {}} aria-label="Global navigation">
+            <BarsIcon />
+          </Button>
+        </MastheadToggle>
+        <MastheadMain>
+          <MastheadBrand>
+            <img src={imgBrand} alt="Patternfly logo" />
+          </MastheadBrand>
+        </MastheadMain>
+        <MastheadContent>
+          <Toolbar id="toolbar">
+            <ToolbarContent>
+              <ToolbarItem>
+                <ContextSelector
+                  toggleText={selected}
+                  onSearchInputChange={this.onSearchInputChange}
+                  isOpen={isOpen}
+                  searchInputValue={searchValue}
+                  onToggle={this.onToggle}
+                  onSelect={this.onSelect}
+                  onSearchButtonClick={this.onSearchButtonClick}
+                  screenReaderLabel="Selected Project:"
+                  isFullHeight
+                >
+                  {filteredItems.map((item, index) => (
+                    <ContextSelectorItem key={index}>{item}</ContextSelectorItem>
+                  ))}
+                </ContextSelector>
+              </ToolbarItem>
+              <ToolbarGroup alignment={{ default: 'alignRight' }}>
+                <ToolbarItem visibility={{ default: 'hidden', lg: 'visible' }}>
+                  <Dropdown
+                    onSelect={this.onDropdownSelect}
+                    toggle={
+                      <DropdownToggle id="toggle-id" onToggle={this.onDropdownToggle} toggleIndicator={CaretDownIcon}>
+                        Dropdown
+                      </DropdownToggle>
+                    }
+                    isOpen={isDropdownOpen}
+                    dropdownItems={dropdownItems}
+                    isFullHeight
```

Comment:
Don't think this would be `isFullHeight`

Reply:
The HTML demo has the dropdown with `pf-m-full-height` though?

Reply:
Yes, you're right. I should change that to [dropdown with image and text](https://www.patternfly.org/v4/components/dropdown/html#dropdown-with-image-and-text). [Issue 4320](https://github.com/patternfly/patternfly/issues/4320)